### PR TITLE
Ewlj 71 homepage prototype

### PR DESCRIPTION
**Pull request by aCyborg**
## Ticket(s)

**Github Issue**

N/A

**Jira Ticket**

- [EWLj-71: Homepage Prototype](https://issues.ama-assn.org/browse/EWLJ-71)

## Description

Creates a homepage prototype with componets:
- Featured Case Organism
- Poll--structured organism variant
- Topics organism
- CTA Band Organism
- About Organism
- Homepage

## To Test

- [ ] Spin up Patternlab
- [ ] Featured Case Organism
- [ ] Poll--structured organism variant
- [ ] Topics organism
- [ ] CTA Band Organism
- [ ] About Organism
- [ ] Homepage

## Visual Regressions
N/A

## Relevant Screenshots/GIFs
N/A

## Remaining Tasks
N/A

## Additional Notes
N/A

### CODE REVIEW

File: styleguide/source/_patterns/03-organisms/current-issue-teaser.twig [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="joe__issue-teaser--left">
```
Comment (gleroux02) [inline styleguide/source/_patterns/03-organisms/current-issue-teaser.twig:2]: Is the month / current issue text supposed to be on the same line as the issue title? 
image

**Review by gleroux02 — COMMENTED** [review]
@aCyborg just one comment for you

**Comment by aCyborg** [comment]
@gleroux02 fixed the issue you were seeing.

**Review by gleroux02 — APPROVED** [review]
👍

**Pull request merged**